File: atcoder/abc/024/d.rb
# -*- coding: utf-8 -*-

i = gets.chomp.to_i
j = gets.chomp.to_i



def get_value(i,j)
  if i == 1 || j == 1
    return 1
  else
    return get_value(i,j-1) + get_value(i-1,j)
  end
end

p get_value(i,j)

File: atcoder/abc/024/d.py
# -*- coding: utf-8 -*-


A = int(input()) % 1000000007
B = int(input()) % 1000000007
C = int(input()) % 1000000007

r = (B*C - A*C) / (A*B - B*C + A*C)
c = (B*C - A*B) / (A*C - B*C + A*B)

print("%d %d" % (r, c))
